src/Header/Header.tsx

```diff
@@ -44,17 +60,29 @@ const WeekButton = () => {
   );
 };
 
-const ArrowLeftButton = () => {
+const ArrowLeftButton = ({ date, setCurrentDate }: HeaderProps) => {
+  const handlePrevWeek = () => {
+    setCurrentDate(addDays(date, -7));
+  };
   return (
-    <button className={classNames(styles.headerButton, styles.noBorder)}>
+    <button
+      className={classNames(styles.headerButton, styles.noBorder)}
+      onClick={handlePrevWeek}
```

Comment:
```suggestion
      onClick={handlePreviousWeek}
```
or we have the convention just to have:
onChange={handleChange}
onClick={handleClick}
onBlur={handleBlur}
...